src/test/java/com/nerodesk/om/aws/AwsUserTest.java

```diff
@@ -53,7 +62,10 @@
     @Test
     public void obtainsDocs() throws Exception {
         MatcherAssert.assertThat(
-            new AwsUser(Mockito.mock(Bucket.class), "user1").docs(),
+            new AwsUser(
+                new MkBucket(this.folder.newFile(), "1"),
```

Comment:
"user1" could be placed at this line according to http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#indentation (last paragraph)